docs/user_manual/source/instruction_fetch.rst

```diff
@@ -27,23 +27,25 @@ are possible and thus it needs fewer signals.
-  +-------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
-  | ``instr_addr_o[31:0]``  | output          | Address, word aligned                                                                                                                             |
-  +-------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
-  | ``instr_prot_o[3:0]``   | output          | Protocol attributes (cacheable, bufferable, etc.)                                                                                                 |
-  +-------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
-  | ``instr_rvalid_i``      | input           | ``instr_rdata_i`` and ``instr_err_i`` are valid when ``instr_rvalid_i`` is high. This signal will be high for exactly one cycle per request.      |
-  +-------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
-  | ``instr_rdata_i[31:0]`` | input           | Data read from memory                                                                                                                             |
-  +-------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
-  | ``instr_err_i``         | input           | An instruction interface error occurred                                                                                                           |
-  +-------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | **Signal**                 | **Direction**   | **Description**                                                                                                                                   |
+  +----------------------------+-----------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+  | ``instr_req_o``            | output          | Request valid, will stay high until ``instr_gnt_i`` is high for one cycle                                                                         |
```

Comment:
Should the instr_reqpar_o also be in the table? Same for all other *_par signals

Reply:
I know I left these out (also didn't upload the new OBI spec yet); will add a preliminary description.